DSGVO SQL show and delete.sql

```diff
@@ -0,0 +1,10 @@
+* Auswahl Anzeige eines Kunden per Nachname
+
+SELECT * 
+FROM Kunde 
```

Comment:
SQL is case sensetive. Table is called KUNDE and not Kunde

Reply:
same as before